R/morse.R

```diff
@@ -308,30 +308,52 @@ NULL
 NULL
 
 #' Survival datasets for \emph{Gammarus pulex} exposed to propiconazole
-#' during 4 days with constant exposure concentration and 10 days with time variable
+#' during 10 days with time variable
 #' exposure concentration (non-standard pulsed toxicity experiment)
 #'
 #' Survival datasets of laboratory bioassays with \emph{Gammarus pulex} 
 #' freshwater invertebrate exposed to several profiles of concentrations
-#' of one fungicide (propiconazole) (constant & pulsed). 
+#' of one fungicide (propiconazole). 
 #'
 #'
-#' @name propiconazole_varC_conc
+#' @name propiconazole_pulse_exposure_conc
```

Comment:
Instead of duplicating the comments on the separated datasets, wouldn't it be better to say explicitly that `propiconazole_pulse_exposure_conc` and `propiconazole_pulse_exposure_conc_Nsurv` are extracted from `propiconazole_pulse_exposure`?